packages/editor/src/components/widgets/code-editor/ScriptInput.tsx

```diff
@@ -12,8 +12,9 @@ const ScriptInput = ({
   editorOptions,
   keyActions,
   modifyAction,
+  browsers,
   ...props
-}: CodeEditorInputProps & { type: string }) => {
+}: CodeEditorInputProps & { type: string; browsers: BrowserType[] }) => {
```

Comment:
I think you can add the browsers attribute to the CodeEditorInputProps type right? Then you don't have to to this on the macro and the script input.